internal/provider/data_source_gitlab_project_tag.go

```diff
@@ -0,0 +1,86 @@
+package provider
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+var _ = registerDataSource("gitlab_project_tag", func() *schema.Resource {
+	return &schema.Resource{
+		Description: "Provide details about a gitlab project tag",
```

Comment:
```suggestion
		Description: `Provide details about a gitlab project tag
				
**Upstream API** : [GitLab API docs](https://docs.gitlab.com/ee/api/tags.html)`,
```

Once applied the docs need to be re-generated.

Reply:
I've already pushed this change ;)